assemble/bin/accumulo-cluster

```diff
@@ -128,7 +128,7 @@ function start_service() {
 function start_tservers() {
   echo -n "Starting tablet servers ..."
   count=1
-  for server in $(egrep -v '(^#|^\s*$)' "${conf}/tservers"); do
+ grep -E -v '(^#|^\s*$)' < "${conf}/tservers" |  while read -r server; do
```

Comment:
The indentation here doesn't look right. Also, you don't need to read in the file with `<` to read it in as STDIN, because grep itself takes file names as arguments. So, you can just do:

```suggestion
  grep -Ev '(^#|^\s*$)' "${conf}/tservers" | while read -r server; do
```